settings.gradle

```diff
@@ -12,4 +12,4 @@ plugins {
 
 rootProject.name = 'coppercore'
 
-include('geometry', 'wpilib_interface', 'math', 'controls', "parameter_tools", 'vision', 'monitors')
+include(':geometry', ':wpilib_interface', ':math', ':controls', ":parameter_tools", ':vision', ':monitors')
```

Comment:
why did we do this? Could this be the source of the build issues?

Reply:
this was changed attempting to fix the build issues, this is what gradle docs say to do for subprojects. It made no difference to the build failing or not though. build currently works with this